docs/api/app.md

```diff
@@ -1064,18 +1064,24 @@ details. Disabled by default.
 
 **Note:** Rendering accessibility tree can significantly affect the performance of your app. It should not be enabled by default.
 
-### `app.setAboutPanelOptions(options)` _macOS_
+### `app.showAboutPanel` _macOS_ _Linux_
+
+Show the app's about panel options. These options can be overridden with `app.setAboutPanelOptions(options)`.
+
+### `app.setAboutPanelOptions(options)` _macOS_ _Linux_
 
 * `options` Object
   * `applicationName` String (optional) - The app's name.
   * `applicationVersion` String (optional) - The app's version.
   * `copyright` String (optional) - Copyright information.
   * `credits` String (optional) - Credit information.
-  * `version` String (optional) - The app's build version number.
+  * `version` String (optional) - The app's build version number. _macOS_
 
 Set the about panel options. This will override the values defined in the app's
 `.plist` file. See the [Apple docs][about-panel-options] for more details.
 
+All except `version` are **required** on Linux.
```

Comment:
On Linux, `applicationName` and `applicationVersion` can just default to what's in package.json.

Reply:
I noticed `credits` is not implemented on Linux yet. If this is intentional, it needs to be documented.

Reply:
On Linux, `copyright` could default to `author` or `author.name` in package.json.

Reply:
oh good point, ty! now i just need to determine how to grab package.json values from c++ 😅